nvtabular/workflow/workflow.py

```diff
@@ -390,6 +389,17 @@ def _clear_worker_cache(self):
         else:
             clean_worker_cache()
 
+    def _zero_output_schemas(self):
+        """
+        Zero out all schemas in order to rerun fit schema after operators
+        have run fit and have stats to add to schema.
+        """
+        for node in iter_nodes(self.output_node.parents_with_dep_nodes):
```

Comment:
Would `iter_nodes([output_node])` not work here? Seems like we should be able to iterate through all the nodes in the loop without having to handle `output_node` as a special case here.